docs/pull_request_template.md

```diff
@@ -0,0 +1,9 @@
+Closes #(number) - (issue name)
+
+This PR introduces... (write a simple description of the added feature or solution to the bug)
+- you can have a bullet list with distinct bits of functionality
+- a good thing to mention here would be API calls made or DB tables accessed
+
+If necessary, describe what any new files are for
+
+@mention your team
```

Comment:
Probably not important to mention your team, you should request them if they are needed to do the code review.

Reply:
Could also add something about testing like **Describe any testing done** as a heading